scripts/classroom-tool

```diff
@@ -1,14 +1,18 @@
 #!/usr/bin/env python
 import os
+import sys
 import github  # Install pygithub
 import git     # Install gitpython
 from argparse import ArgumentParser
 import datetime
+import base64
+import json
 from configparser import ConfigParser
 import logging
 import pandas as pd
 from functools import cache
 import time
+from selenium import webdriver
```

Comment:
Needs adding as a dependency in setup.cfg

Reply:
Done.